package.json

```diff
@@ -5,7 +5,8 @@
   "main": "server.js",
   "scripts": {
     "test": "jest",
-    "dev": "nodemon server.js"
+    "dev-server": "nodemon server.js",
+    "start": "pm2 start server.js"
   },
```

Comment:
why do you use an npm script to run pm2? pm2 is an alternative way to run your node server, it does not need npm.

Reply:
For consistency - it doesn't need it but if you start everything else with npm it's just easy. Also npm start is a very standard way to start a server so if someone else cloned my server they could just use npm start like they are used to and not have to worry about what's under the hood.